spec/integration/admin_orders_spec.rb

```diff
@@ -65,4 +75,12 @@ def mock_completed_order
                                     status:  "completed",
                                     total_price: 146789)
   end
+
+  def mock_seven_paid_orders
+    7.times do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.